specification/cognitiveservices/data-plane/ComputerVision/readme.md

```diff
@@ -0,0 +1,33 @@
+# Cognitive Services Computer Vision SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating Computer Vision SDK.
+
+The current release is `release_1_0`.
+
+``` yaml
+
+tag: release_1_0
+openapi-type: data-plane
+```
+# Releases
+
+### Release 1.0
+These settings apply only when `--tag=release_1_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_1_0'
+input-file: v1.0/ComputerVision.json
+```
+
+## CSharp Settings
+These settings apply only when `--csharp` is specified on the command line.
+``` yaml $(csharp) 
+csharp: 
+  license-header: MICROSOFT_MIT_NO_VERSION
+  azure-arm: false
+  namespace: Microsoft.Azure.CognitiveServices.Vision.ComputerVision
+  output-folder: $(csharp-sdks-folder)/CognitiveServices/dataPlane/Vision/Vision/Generated/ComputerVision
```

Comment:
did you mean to have `Vision/Vision` in the path?

Reply:
Yes, first Vision is the solution folder, second Vision is the project folder. Having full namespace/package name in the path will push path length limitation.